setup.py

```diff
@@ -16,10 +16,10 @@ def find_stub_files(name):
     return result
 
 
-with open("README.md", "r") as f:
+with open("README.md") as f:
     readme = f.read()
 
-dependencies = ["mypy>=0.782", "django-stubs>=1.6.0", "typing-extensions", "requests>=2.0.0", "coreapi>=2.0.0"]
+dependencies = ["mypy>=0.782", "django-stubs>=1.6.0", "typing-extensions>=3.7.2", "requests>=2.0.0", "coreapi>=2.0.0"]
```

Comment:
Let's bump `mypy` to `0.790`

Reply:
Will do

Reply:
so this is not currently possible because of this:
```
ERROR: django-stubs 1.6.0 has requirement mypy<0.790,>=0.782, but you'll have mypy 0.790 which is incompatible.
```
So I'll just keep it at >=0.782

Reply:
ok